lib/assessment.rb

```diff
@@ -20,20 +30,34 @@
 # Name and location should be writeable, but age should be read-only.
 
 ##
-# your code here
+class Person
+
+  attr_writer :name, :location
```

Comment:
Missed this earlier, but you should be using `attr_accessor` here, not `attr_writer`. `attr_writer` is almost never used because it gives us a 'write' method without giving us a 'read' one, and it's very unusual to be able to change something's value but not be able to see what it is.